includes/Modules/Tag_Manager.php

```diff
@@ -112,7 +112,7 @@ public function register() {
 	 */
 	public function get_scopes() {
 		return array(
-			'https://www.googleapis.com/auth/tagmanager.readonly',
+			Google_Service_TagManager::TAGMANAGER_READONLY,
```

Comment:
Ah I see you did it here as well. Let's keep it using the literal for now and then open a new issue to refactor with the constants for all modules 👍